test/config/integration/certs/expired_cert.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Don't these files get generated by a script? Can you modify the script?

Reply:
Sure.

Reply:
Sorry about force push for DCO fix.